### ISSUE TITLE
Multilanguage support

### ISSUE THREAD

**Issue by FelixINX** [LPardue/SpotifyStatusApplet#1]
Hi,
Do you think that translate the applet is possible, or we need to make another applet?

Also, why there are no `app.manifest`?

**Comment by LPardue** [comment]
Hi,

Thank you for your questions. 

I think translation is possible, there are not too many strings to convert. Did you have any particular language in mind?

I will look into the app.manifest issue.

My dev environment is in transit at the moment, so there may be a delay in turn around on these issues. I'll aim to take a look on the weekend.

**Comment by FelixINX** [comment]
Yes, it's possible, i'm sure. I have started, but I can't build the program because of the missing `app.manifest`. My translation is in french.

I'm building from VS Community 2015.
